tests/lava/proc/conv/test_models.py

```diff
@@ -0,0 +1,304 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+import unittest
+import numpy as np
+import sys
+
+from lava.magma.core.decorator import implements, requires, tag
+from lava.magma.core.model.py.model import PyLoihiProcessModel
+from lava.magma.core.model.py.ports import PyOutPort, PyInPort
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.process.ports.ports import OutPort, InPort
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.core.process.variable import Var
+from lava.magma.core.resources import CPU
+from lava.magma.core.run_configs import RunConfig
+from lava.magma.core.run_conditions import RunSteps
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+from lava.proc.conv.process import Conv
+from lava.proc.conv import utils
+
+
+verbose = True if (('-v' in sys.argv) or ('--verbose' in sys.argv)) else False
+TORCH_IS_AVAILABLE = utils.TORCH_IS_AVAILABLE
+
+
+class ConvRunConfig(RunConfig):
+    """Run configuration selects appropriate Conv ProcessModel based on tag:
+    floating point precision or Loihi bit-accurate fixed point precision"""
+    def __init__(self, custom_sync_domains=None, select_tag='fixed_pt'):
+        super().__init__(custom_sync_domains=custom_sync_domains)
+        self.select_tag = select_tag
+
+    def select(self, proc, proc_models):
+        for pm in proc_models:
+            if self.select_tag in pm.tags:
+                return pm
+        raise AssertionError('No legal ProcessModel found.')
+
+
+class SendProcess(AbstractProcess):
```

Comment:
Didn't we want to factor such shared collateral out such that many processes can use it? 

If you are the first to need/write it, then start by putting it into a shared place.

Reply:
This could even be more than just testing infrastructure and go into the io process lib.

Reply:
sure I will move them to lava.proc.io